cluster/charts/crossplane/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- name: oam-kubernetes-runtime
+  version: 0.1.1-1.g8cde76d
```

Comment:
There are issues with how the `oam-kubernetes-runtime` chart is being promoted on official releases (https://github.com/crossplane/oam-kubernetes-runtime/issues/226), so we will start with it just pinned to the latest version.

Reply:
just curious, is this `requirements.yaml` identified by the helm automatically as the dependencies or you can put the `dependencies` field anywhere in the directory?

Reply:
@ryanzhang-oss we are currently building with helm 2, which requires using `requirements.yaml`. Helm 3 allows you to optionally just put them in-line in the `Chart.yaml` :+1: